auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -190,13 +375,41 @@
                   <span class="open-action">
                     <v-btn
                       color="primary"
-                      :class="['open-action-btn', getButtonLabel(item.status).toLowerCase()]"
+                      :class="['open-action-btn', { active: item.nrNumber }]"
```

Comment:
I don't understand this syntax. What does the right hand side evaluate to? Should be True or False.

Reply:
When the item has a nrNumber, We will adjust the styles of the "Open" Button:
![image](https://github.com/user-attachments/assets/999451c6-9b5a-4752-9511-41a9d2a78bf6)

Reply:
Please change this to the usual syntax:
```
   class="open-action-btn"
   :class="{ 'active': item.nrNumber }"
```

Reply:
Updated

Reply:
Looks good. Resolving.